oag/ja-jp/content/topics/access-gateway/advanced-oag-policy-development.htm

```diff
@@ -318,17 +318,17 @@ <h3>高度なポリシーを定義する</h3>
                                                             <ol>
                                                                 <li value="1"><span class="mc-variable okta-feature-names.Access Gateway_UI variable">Access Gateway 管理者 UI コンソール</span>にサインインします。</li>
                                                                 <li value="2"><span class="uicontrol">[Application（アプリケーション）]</span>タブをクリックします。</li>
-                                                                <li value="3"><span class="uicontrol">[Edit（編集）]</span>をクリックします。<br /><br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.01.png" data-mc-popup-alt="Click the pencil icon next to any application to open for update."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="763" data-mc-height="300" src="../../resources/images/access-gateway/admin-policy-guide-advanced.01_thumb_220_220.png" alt="更新するアプリケーションの横にある鉛筆アイコンをクリックします。" tabindex="" id="imgimg_new" /></a><br /></li>
+                                                                <li value="3"><span class="uicontrol">［Edit（編集）］</span>をクリックします。<br /><br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.01.png" data-mc-popup-alt="Click the pencil icon next to any application to open for update."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="763" data-mc-height="300" src="../../resources/images/access-gateway/admin-policy-guide-advanced.01_thumb_220_220.png" alt="更新するアプリケーションの横にある鉛筆アイコンをクリックします。" tabindex="" id="imgimg_new" /></a><br /></li>
                                                                 <li value="4"><span class="uicontrol">[Policies（ポリシー）]</span>をクリックします。<br /></li>
-                                                                <li value="5"><span class="uicontrol">[Edit（編集）]</span>をクリックします。<br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.03.png" data-mc-popup-alt="Click the pencil to open a policy for edit."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="764" data-mc-height="123" src="../../resources/images/access-gateway/admin-policy-guide-advanced.03_thumb_220_220.png" alt="鉛筆アイコンをクリックして編集対象のポリシーを開きます。" tabindex="" id="imgimg_new" /></a></li>
+                                                                <li value="5"><span class="uicontrol">[編集]</span>をクリックします。<br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.03.png" data-mc-popup-alt="Click the pencil to open a policy for edit."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="764" data-mc-height="123" src="../../resources/images/access-gateway/admin-policy-guide-advanced.03_thumb_220_220.png" alt="鉛筆アイコンをクリックして編集対象のポリシーを開きます。" tabindex="" id="imgimg_new" /></a></li>
```

Comment:
@pengyao-okta I found a regression here. This seems to be a one-off, there are several other topics in this PR that have the correct rendering of UI labels.